spec/grid_spec.rb

```diff
@@ -73,14 +73,23 @@
       expect(grid.coordinate_valid?(invalid_lower_case)).to eq false
     end
   end
-  describe "#mark" do
-    it "takes in a valid coordinate, and gives it a marked value in coordinates" do
-      coordinate = "A3"
-      grid = Grid.new
 
-      grid.mark(coordinate)
+  it "#mark takes in a valid coordinate, and gives it a marked value in coordinates" do
```

Comment:
This description implies that it will do something different when the coordinate is invalid, so I would expect to see an example for a bad coordinate too. But I think maybe the description is just off.

Reply:
✔️